napari/_tests/test_advanced.py

```diff
@@ -180,6 +180,34 @@ def test_update_console(make_napari_viewer):
         del viewer.window._qt_viewer.console.shell.user_ns[k]
 
 
+@pytest.mark.filterwarnings("ignore::DeprecationWarning:jupyter_client")
+def test_update_lazy_console(make_napari_viewer):
```

Comment:
I am trying to understand if this test will still do its job if the startup time for console is decreased to the point that it is loaded when `update_console` is called.

Reply:
With the [lazy instantiation](https://github.com/napari/napari/blob/b5696770d31cbaa6dc02068b8b8b0a95bdd8a107/napari/_qt/qt_viewer.py#L489) the console shouldn't begin starting up until the assertion on L196 below.